tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
+        prop_file = ""
+        self.security_config = self.kafka.security_config.client_config(self.prop_file)
```

Comment:
It seems problematic to pass in `self.prop_file` here `self.kafka.security_config.client_config(self.prop_file)`

since `self.prop_file` is now a function and not a string